apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.tsx

```diff
@@ -54,8 +60,10 @@ function getTourAssets( key: string ): TourAsset | undefined {
 function getTourSteps(
 	localeSlug: string,
 	referencePositioning = false,
-	isSiteEditor = false
+	isSiteEditor = false,
+	themeName: string | null = null
 ): WpcomStep[] {
+	const isVideoMaker = [ 'videomaker', 'videomaker white' ].indexOf( themeName ) >= 0;
```

Comment:
I saw a cute (ES6?) solution once that read something like: 

```
[ 'videomaker', 'videomaker white' ].some( videomakerTheme => themeName.includes( videomakerTheme ) );
```

Your works the same though ;)

Reply:
Seems like this one would be even shorter, while keeping the same precision:

```suggestion
	const isVideoMaker = [ 'videomaker', 'videomaker white' ].includes( themeName );
```

Reply:
Thanks, using `includes` looks much cleaner!